src/classes/services/WebSocketService.ts

```diff
@@ -4,23 +4,20 @@ import MessageCommands from '@/types/MessageCommands'
 import MessageTypes from '@/types/MessageTypes'
 import File from '@/types/File'
 import base64ToArrayBuffer from '@/utils/helpers/base64ToArrayBuffer'
-import isDownloadable from '@/utils/helpers/isDownloadable'
 import { useRouter } from 'vue-router'
-import {Buffer} from 'buffer'
-//Open WebSockets
+import { Buffer } from 'buffer'
 
 export class webSocketService {
   private ws = {} as WebSocket
   private shouldDisconnect = false
   private readonly router = useRouter()
   listFiles = [] as File[]
-  // private readonly wsOnmessageListenersAuth: Array<() => void>
-  private wsOnMessageListenersListFiles: ((listfiles: any) => void) | null =
+
+  private wsOnMessageListenersListFiles: ((listfiles: File[]) => void) | null =
     null
   constructor() {
     console.log('Starting connection to WebSocket Server')
     this.ws = new WebSocket('wss://cloudon.cc:9292/')
-    // this.wsOnmessageListenersAuth = []
 
     this.ws.onopen = () => {
```

Comment:
Can we move it away to the main body of `WebSocketService`?

Reply:
It's a better practice to set up the WebSocket connection and event handlers in the constructor of the class so that it's only done once when the class is instantiated, and the same instance can be reused throughout the lifetime of the application.